libs/wingsdk/src/target-tf-aws/api.ts

```diff
@@ -390,70 +390,105 @@ class WingRestApi extends Construct {
     super(scope, id);
     this.region = (App.of(this) as App).region;
 
-    const defaultResponse = API_CORS_DEFAULT_RESPONSE(props.cors);
+    // const defaultResponse = API_CORS_DEFAULT_RESPONSE(props.cors);
+
+    // let CORS = true;
+
+    let OAS30: OpenApiSpec = {
+      openapi: "3.0.1",
+      info: {
+        title: "fugazi",
+        version: "v1",
+      },
+      paths: {
+        "/$default": {
+          "x-amazon-apigateway-any-method": {
+            isDefaultRoute: true,
+            "x-amazon-apigateway-integration": {
+              payloadFormatVersion: "1.0",
+              type: "http_proxy",
+              httpMethod: "ANY",
+              uri: "https://example.com/",
+              connectionType: "INTERNET",
+            },
+          },
+        },
+      },
+      "x-amazon-apigateway-importexport-version": "1.0",
+    };
+
+    // let OAS30 = CORS
+    //   ? {
+    //       ...OAS30_BASE,
+    //       "x-amazon-apigateway-cors": {
+    //         allowMethods: ["GET", "OPTIONS", "POST"],
+    //         allowHeaders: ["*"],
+    //         maxAge: 0,
+    //         allowCredentials: false,
+    //         allowOrigins: ["*"],
+    //       },
+    //     }
+    //   : OAS30_BASE;
+
+    const apigwCorsConfig = props.cors
+      ? {
+          corsConfiguration: {
+            // renamed allowOrigin to allowOrigins
+            allowOrigins: props.cors.allowOrigins
```

Comment:
I also found it weird that AWS renamed the header, but I still prefered to have matching names.

Reply:
Given that Wing is intending to abstract the cloud resources, I wouldn't follow whatever the AWS platform is doing by default. At the moment there's also the `local` target and in the future gcp, azure and potentially more targets for the API resource

Other than that, since `allowOrigin` is already released, this would be a breaking change. Doesn't mean this can't change, but don't see a strong case to support this at the moment.

Reply:
Ok, I named it back to `allowOrigin`.